aspnetcore/mvc/views/razor.md

```diff
@@ -15,74 +15,74 @@ uid: mvc/views/razor
 
 By [Taylor Mullen](https://twitter.com/ntaylormullen) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-## What is Razor?
-
-Razor is a markup syntax for embedding server based code into web pages. The Razor syntax consists of Razor markup, C# and HTML. Files containing Razor generally have a *.cshtml* file extension.
+Razor is a markup syntax for embedding server-based code into webpages. The Razor syntax consists of Razor markup, C#, and HTML. Files containing Razor generally have a *.cshtml* file extension.
 
 ## Rendering HTML
 
-The default Razor language is HTML. Rendering HTML from Razor is no different than in an HTML file. A Razor file with the following markup:
+The default Razor language is HTML. Rendering HTML from Razor markup is no different than rendering HTML from an HTML file. If you place the following markup into a Razor file, it's rendered by the server unchanged:
```

Comment:
If you're modifying this paragraph to state that bits are unchanged I'm not convinced you need to provide any example below